packages/analytics-plugin-google-analytics/src/node.js

```diff
@@ -7,12 +7,24 @@ if (!process.browser) {
 /**
  * Serverside Google Analytics plugin
  * @param {object} pluginConfig - Plugin settings
- * @param {string} pluginConfig.trackingId - Google Analytics site tracking Id
+ * @param {string} pluginConfig.trackingId - Google Analytics site tracking Id. Same as tid (se below), only one needed
+ * @param {string} pluginConfig.tid - Google Analytics site tracking Id. Same as trackingId (se above), only one needed
+ * @param {string} [pluginConfig.cid] - Google Analytics client Id. It generates a random UUID if none is set
+ * @param {boolean} [pluginConfig.strictCidFormat] - Set it to false to set a custom client Id (not UUID). Only used if cid is set
```

Comment:
It doesn't look like `strictCidFormat` is used. Do we need that?

Reply:
Yes, you need to set `strictCidFormat` to `false` if you want to use a non UUID `clientId`. It's explained in the [readme](https://github.com/peaksandpies/universal-analytics/blob/master/README.md) file and also you can see [here](https://github.com/peaksandpies/universal-analytics/blob/master/lib/index.js#L57) how it's used.